src/naming-strategy/DefaultNamingStrategy.ts

```diff
@@ -60,7 +60,7 @@ export class DefaultNamingStrategy implements NamingStrategyInterface {
         return columnName + "_" + index;
     }
 
-    joinTableColumnName(tableName: string, propertyName: string, columnName?: string): string {
+    joinTableColumnName(tableName: string, propertyName: string, columnName?: string, inverse?: boolean): string {
```

Comment:
Maybe its better to create separate `joinTableInverseColumnName` method instead?

Reply:
That would definitely be more transparent, I just did not want to introduce new methods. But if that is OK with you then I am going to change it.

Reply:
yeah lets change it